source/main/gui/panels/GUI_GameMainMenu.cpp

```diff
@@ -183,3 +181,7 @@ void GameMainMenu::DrawVersionBox()
     ImGui::PopStyleColor(1);
 }
 
+bool GameMainMenu::HighlightButton(const std::string& txt,ImVec2 btn_size, int index) const{
+    std::string button_txt = (m_kb_focus_index == index) ? fmt::format("--> {}  <--", txt) : txt;
```

Comment:
the caption is de-centered (1space on left, 2 on right). Otherwise, :heart: this function